README.md

```diff
@@ -16,7 +16,9 @@ Below is a sample `settings.json` file to be included in the root of your Meteor
 			"debug_data": true,
 			"ddp_url": "http://drupalddp.dev",
 			"restws_user": "restws_xxxxx",
-			"restws_pass": "your_password"
+			"restws_pass": "your_password",
+			"simple_security": true,
+			"simple_security_token": "kZtVE4rsE@U3#UD3tgxy5F@QZfjETM"

```

Comment:
I think we should not provide a default token and throw an error if one isn't set. Maybe even prevent Drupal DDP from working if not set. I am afraid some will just use this token and not think about the fact that everyone who pokes around the code will have it.